### PR TITLE
🌟 Added open desktop app when it's a Twake link #2316 

### PR DESCRIPTION
**Pull request by rezk2ll**
## Description

- Added logo atom and its story
- Added link atom story
- Added open desktop popup component
- Added Desktop redirect view

## Related Issue

#2316 

## Motivation and Context

- when a Twake link is opened the web app should open the desktop app if it is installed ( custom protocol is registered )

## How Has This Been Tested?

- locally

## Screenshots (if appropriate):
image

https://user-images.githubusercontent.com/6764881/176155602-0e319fb4-25a4-42f7-b6e2-244d30aaac2e.mp4

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: twake/frontend/src/app/atoms/logo/index.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+type PropsType = {
+  size?: 16 | 32 | 64 | 128 | 256;
+}
+
+export const Logo = ({ size = 128 }: PropsType) => (
+  <div className="flex flex-col items-center justify-center">
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/atoms/logo/index.tsx:8]: A good practice is to do this:
`interface PropsType extends Omit<React.HTMLAttributes<HTMLDivElement>, "size">{
  size?: 16 | 32 | 64 | 128 | 256;
}`

and then 

```
  <div className={"flex flex-col items-center justify-center " + props.className} {..._.omit(props, "className", "size")}>
```

File: twake/frontend/src/app/components/open-desktop-popup/open-desktop-popup.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  useEffect(() => {
+    setTimeout(() => {
+      setShowLink(true);
+    }, 5000);
+  }, []);
+
+  return (
+    <div className="flex flex-col items-center justify-center h-full overflow-hidden space-y-2">
+      <Logo />
+      <Subtitle>Opened in Twake app</Subtitle>
```
Comment (RomaricMourgues) [inline twake/frontend/src/app/components/open-desktop-popup/open-desktop-popup.tsx:21]: Let's add mt-4 or mt-8 to the Subtitle className

**Review by RomaricMourgues — CHANGES_REQUESTED** [review]
Nice, just two small feedback 👍

File: twake/frontend/src/utils/detection/detect-firefox.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import { twakeApplicationUrlScheme, wait } from './common';
```
Comment (RomaricMourgues) [inline twake/frontend/src/utils/detection/detect-firefox.ts:1]: Can we rename the folder "detection" to "browser-detect" ?

**Review by RomaricMourgues — CHANGES_REQUESTED** [review]
Just on folder to rename, the rest seems good

File: twake/frontend/.storybook/main.js [hunk 4]
```diff
@@ -66,4 +66,5 @@ module.exports = {
   core: {
     builder: 'webpack5',
   },
+  staticDirs: ['../public'],
```
Comment (RomaricMourgues) [inline twake/frontend/.storybook/main.js:69]: Why did you add this ? (it produce a bug in the tests)

Comment (rezk2ll) [reply twake/frontend/.storybook/main.js]: i removed it from the config file and added it when actually running storybook ( in package.json )

Comment (RomaricMourgues) [reply twake/frontend/.storybook/main.js]: In this case change the package.json with:
```
    "build-storybook": "build-storybook -o tmp_storybook; rm -rf public/storybook; mv tmp_storybook public/storybook",
```

And also add to the git ignore `tmp_storybook`

The tests will pass in this case
